packages/wallet-ui/package.json

```diff
@@ -20,7 +20,7 @@
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "test": "react-scripts test",
+    "test": "CI=true react-scripts test",
```

Comment:
@joelalejandro had to add this because https://github.com/facebook/create-react-app/issues/6899